code/game/turfs/simulated/wall/reinf_walls.dm

```diff
@@ -227,3 +227,30 @@
 /turf/closed/wall/r_wall/rcd_act(mob/user, obj/item/construction/rcd/the_rcd, passed_mode)
 	if(the_rcd.canRturf)
 		return ..()
+
+/turf/closed/wall/r_wall/syndicate
+	name = "hull"
+	desc = "The armored hull of an ominous looking ship."
+	icon = 'icons/turf/walls/plastitanium_wall.dmi'
+	icon_state = "map-shuttle"
+	explosion_block = 20
+	sheet_type = /obj/item/stack/sheet/mineral/plastitanium
+	smooth = SMOOTH_MORE|SMOOTH_DIAGONAL
+	canSmoothWith = list(/turf/closed/wall/r_wall/syndicate, /turf/closed/wall/mineral/plastitanium, /obj/machinery/door/airlock/shuttle, /obj/machinery/door/airlock, /obj/structure/window/plastitanium, /obj/structure/shuttle/engine, /obj/structure/falsewall/plastitanium)
```

Comment:
Instead of just making reskinned r_walls, why not just buff plastitanium walls to be proper explosion proof? I'm a big fan of making as much of the universe craftable/creatable as possible.

Reply:
Is there a problem with buffing plastitanium walls though? Where are they used, apart from the infiltrator?

Reply:
Some examples off the top of my head: Delta station uses them in several places, mainly around the escape pods. A bunch of different shuttles use them as well.

Furthermore, this iteration *just works*. I'm not really interested in bikeshedding over plastitanium walls, and I'm certainly not interested over having to go back through and re-do all the mapping work. I see the value in having both types of walls. It gives a map maker more options. Ideally, like I said, the "Syndie R-walls" would have a different set of sprites, but I'm not skilled enough to sprite it. This is a port of someone else's PR and the primary focus of the PR is to eradicate a particularly silly and arbitrary rule.

Reply:
Ooof, I got told. I withdraw my objections.

Reply:
tfw delta ruins syndicate balloon and now magic walls, what will it hurt next